CHANGELOG-1.1.md

```diff
@@ -1,3 +1,62 @@
+# TiDB Operator v1.1.0-rc.1 Release Notes
+
+This is a release candidate of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Action Required
+
+- `--advertise-address` will be configured for `tidb-server` which will trigger rolling-upgrade for `tidb-server` component, you can set `spec.paused` to `true` to avoid it before upgrading tidb-operator and set it back to `false` when it's ready to upgrade your tidb server. ([#2076](https://github.com/pingcap/tidb-operator/pull/2076), [@cofyc](https://github.com/cofyc))
+- Add field `tlsClient` in the backup and restore spec, which controls whether to enable TLS for connection with TiDB Server and support specifying a secret name that includes the cert. ([#2003](https://github.com/pingcap/tidb-operator/pull/2003), [@shuijing198799](https://github.com/shuijing198799))
```

Comment:
```suggestion
- Add field `tlsClient.tlsSecret` in the backup and restore spec, which supports specifying a secret name that includes the cert. ([#2003](https://github.com/pingcap/tidb-operator/pull/2003), [@shuijing198799](https://github.com/shuijing198799))
```

Reply:
Whether to enable TLS is retrieved from tc.

Reply:
updated